docs/05-staking/03-how_to_stake_on_eon.md

```diff
@@ -0,0 +1,89 @@
+        console.log('Account ' + accounts[0]);
+
+        const abi = require("browser/contracts/artifacts/abi_fs.json"); // Change this for different path
+
+        const contract = new web3.eth.Contract(abi, smartContractAddress, {from: accounts[0]});
+
+        // Testing method:  delegate
+        const ownerAddress = accounts[0];
+        // pick one of the existing forgers, or create a new forger
+        const blockSignPublicKey = "0x" + YOUR_BLOCK_SIGN; // the address that will sign the block when forged; it can be the same address as the ownerAddress or a different one (delegate forging); if this value is not specified the ownerAddress will be used
+        const forgerVrfPublicKey = "0x" + YOUR_VRF_KEY; // this is the key created in the previous step
+        const first32BytesForgerVrfPublicKey = forgerVrfPublicKey.substring(0, 66);
+        const lastByteForgerVrfPublicKey = "0x" + forgerVrfPublicKey.substring(66, 68);
+
+        methodName = 'delegate';
+        console.log('Response for '+ methodName);
+
+        const response = await contract.methods.delegate(blockSignPublicKey,first32BytesForgerVrfPublicKey,lastByteForgerVrfPublicKey,ownerAddress).send({value: amount *10**18}).then(console.log);
+
+    } catch (e) {
+        console.log("Error:" + e.message);
+    }
+  })()
+```
+⚠️ **Please, remember to replace YOUR_BLOCK_SIGN and YOUR_VRF_KEY with the actual keys, and make sure to specify the path of the native contract ABI (not provided in the documentation, but available here together with the complete Remix workspace)!**  
+⚠️ **Please, notice that with this operation you are increasing the voting power of the forger defined by the blockSignPublicKey and forgerVrfKey, and with that the chance to produce blocks. The operation is a delegation of stake but no rewards will be automatically forwarded to you if you are not the owner of both blockSignPublicKey and forgerVrfKey. Double check them before executing the transaction.
+The transaction is reversible by executing a transaction signed by the ownerAddress.**
+
+#### Enable forging
+At last make a `POST` request to `/block/startForging` to command the node to start forge blocks (if the node is not yet running you can set to true the `automaticForging` parameter in the `forger` section of the config file).  
+To stop forging you can call a similar endpoint `/block/stopForging`.
+
+#### Stake maturity
+When a forger stake is created is not immediately ready to let the forger take part into the forgers' election. It needs **2 consensus epochs** before the stake reaches the maturity period; after that, the forger will be ready to take part into the slot leader lottery and be able to forge new blocks. 
+
+#### Verify the stake is created
+To make sure the operations above were successful, you can make a `POST` request to `/transaction/allForgingStakes` and verify the publicKey used to create the forger stake appears in the returned list.
```

Comment:
To make sure the operations was successful, make a `POST` request to `/transaction/allForgingStakes` and verify that the **publicKey** used to create the forger stake appears in the returned list.